broker/src/main/java/io/camunda/zeebe/broker/partitioning/PartitionManager.java

```diff
@@ -22,8 +22,8 @@
 public class PartitionManager {
```

Comment:
:question: Is there a reason not to make this final? If it's for mocking, we can already introduce an interface, as tbh, I expect this will become an interface anyway (though we may not yet know the final API).